lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -1099,16 +1140,20 @@ def select_stickiness_policy(self):
                         'param_value': self.stickiness['cookie']
                     }
                     policy.append(self._policy_name(policy_attrs['type']))
-                    self._set_stickiness_policy(elb_info, listeners_dict, policy, **policy_attrs)
+                    if not self.module.check_mode:
+                        self._set_stickiness_policy(elb_info, listeners_dict, policy, **policy_attrs)
                 elif not self.module.boolean(self.stickiness['enabled']):
                     if len(elb_info.policies.app_cookie_stickiness_policies):
-                        if elb_info.policies.app_cookie_stickiness_policies[0].policy_name == self._policy_name(policy_type):
+                        if elb_info.policies.app_cookie_stickiness_policies[0].policy_name == self._policy_name(
```

Comment:
Not sure why we've wrapped here and 1118, decreasing readability, while not wrapping longer lines.